webapp/channels/src/components/sidebar/sidebar_category_header.tsx

```diff
@@ -40,6 +40,14 @@ type Props = StaticProps & {
 }
 
 export const SidebarCategoryHeader = React.forwardRef((props: Props, ref?: React.Ref<HTMLButtonElement>) => {
+    const {dragHandleProps} = props;
+
+    // (Accessibility) Ensures interactive controls are not nested as they are not always announced
+    // by screen readers or can cause focus problems for assistive technologies.
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    delete dragHandleProps?.role;
```

Comment:
This fixes the issue in the screenshot.
<img width="834" alt="Screenshot 2023-07-26 at 5 52 28 PM" src="https://github.com/mattermost/mattermost/assets/5334504/0af7761a-0b9a-4ee4-90ad-42d55a1e3c1c">